dotnet/src/Connectors/Connectors.OpenAIV2/Core/ClientCore.cs

```diff
@@ -0,0 +1,213 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.ClientModel;
+using System.ClientModel.Primitives;
+using System.Collections.Generic;
+using System.Net.Http;
+using System.Runtime.CompilerServices;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using Microsoft.Extensions.Logging.Abstractions;
+using Microsoft.SemanticKernel.Http;
+using OpenAI;
+using OpenAI.Embeddings;
+
+#pragma warning disable CA2208 // Instantiate argument exceptions correctly
+
+namespace Microsoft.SemanticKernel.Connectors.OpenAI;
+
+/// <summary>
+/// Base class for AI clients that provides common functionality for interacting with OpenAI services.
+/// </summary>
+internal class ClientCore
+{
+    /// <summary>
+    /// Model Id or Deployment Name
+    /// </summary>
+    internal string ModelName { get; set; } = string.Empty;
```

Comment:
Given this is OpenAI specific do we still need to make this Model Id or Deployment Name?

Reply:
This was ModelIDOrDeploymentName in the original CLientCore, and I already change the name just to ModelId, as this ClientCore is just for OpenAI, I also merged the original OpenAIClientCore with this one for the same reason.

Reply:
Can you update the comment and property name so.